activestorage/app/models/active_storage/blob.rb

```diff
@@ -14,13 +14,19 @@
 # update a blob's metadata on a subsequent pass, but you should not update the key or change the uploaded file.
 # If you need to create a derivative or otherwise change the blob, simply create a new blob and purge the old one.
 class ActiveStorage::Blob < ActiveRecord::Base
+  class UnpreviewableError < StandardError; end
+
   self.table_name = "active_storage_blobs"
 
   has_secure_token :key
   store :metadata, coder: JSON
 
   class_attribute :service
 
+  class_attribute :default_url_expiry, default: 5.minutes
```

Comment:
Don't think we often have configs that include `default` in the name. How about `service_url_expiry`? Or maybe `service_url_expires_in` since that connotes the relativeness of it and that users would want relative timestamps.

Alternatively, should this be a reader that wraps an `ActiveStorage.service_url_expiry`? (Which should then go through `config.active_storage.service_url_expiry` like @matthewd mentioned).